build/Dockerfile

```diff
@@ -63,6 +63,11 @@ WORKDIR /build/windows-machine-config-operator/cloud-provider-azure/
 COPY cloud-provider-azure/ .
 RUN GOOS=windows go build -o azure-cloud-node-manager.exe ./cmd/cloud-node-manager
 
+# Build ecr-credential-provider
+WORKDIR /build/windows-machine-config-operator/cloud-provider-aws/
+COPY cloud-provider-aws/ .
+RUN env -u VERSION GOOS=windows make ecr-credential-provider
```

Comment:
Have we explored the cross-build make target, does it help us with getting the extensions we need for Windows? https://github.com/openshift/cloud-provider-aws/blob/51180c32ceb70298faad1aac1022d8983fdf2e78/Makefile#L208